frontend/src/components/Badges.tsx

```diff
@@ -1,33 +1,22 @@
 import './Badges.css';
 
 export interface IBadgesProps {
-    discordImages: string[][]; // in array pos 0 is img pos 1 is description
-    redditImages: string[][];
+    images: string[][] //pos 0 is image, pos 1 is name. will probably be a more complex obj once I get api data.
```

Comment:
Instead of a nested array, what about a map here from name to image? Is that possible  in TS?